whois-nrtm4-client/src/test/java/net/ripe/db/nrtm4/client/processor/UNFProcessorWrongSignatureTestIntegration.java

```diff
@@ -0,0 +1,35 @@
+package net.ripe.db.nrtm4.client.processor;
+
+import net.ripe.db.nrtm4.client.AbstractNrtmClientIntegrationTest;
+import net.ripe.db.nrtm4.client.dao.NrtmClientVersionInfo;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+@Tag("IntegrationTest")
+public class UNFProcessorWrongSignatureTestIntegration extends AbstractNrtmClientIntegrationTest {
+
+    @BeforeAll
```

Comment:
I need to create a new IT because the property is different. Here I want to test that the signature if wrong
You told me already that if I want to change the property I need to create a different IT

Reply:
Can you instead return a UNF signed with different key from dummy server instead ? So it will be failed ?

Reply:
Done. It doesn't fail but I does nothing. Just return null and add a log
I prefer this because the client will then try again in 1 minute

Reply:
I meant so verification failed :) and you should throw an exception as well and it is a scheduled job so it will be run again  next min anyways

Reply:
If I throw and exception is get blocked by shedlock :(

Reply:
ok, there a bug with shedlock then. thanks leave it as it is then